pkg/apis/pipeline/v1/pipelinerun_defaults.go

```diff
@@ -37,8 +37,14 @@ func (pr *PipelineRun) SetDefaults(ctx context.Context) {
 func (prs *PipelineRunSpec) SetDefaults(ctx context.Context) {
 	cfg := config.FromContextOrDefaults(ctx)
 
-	if prs.Timeouts != nil && prs.Timeouts.Pipeline == nil {
-		prs.Timeouts.Pipeline = &metav1.Duration{Duration: time.Duration(cfg.Defaults.DefaultTimeoutMinutes) * time.Minute}
+	if prs.Timeouts == nil {
+		prs.Timeouts = &TimeoutFields{}
+	}
```

Comment:
this block can be removed now

Reply:
ehhh sorry it's restored on the remote branch..